app/components/dashboard/collection_header_component.rb

```diff
@@ -0,0 +1,27 @@
+# typed: false
+# frozen_string_literal: true
+
+module Dashboard
+  # Renders a header for a summary table
+  class CollectionHeaderComponent < ApplicationComponent
+    sig { params(collection: Collection).void }
+    def initialize(collection:)
+      @collection = collection
+    end
+
+    attr_reader :collection
+
+    delegate :allowed_to?, to: :helpers
+    delegate :depositing?, to: :collection
+
+    sig { returns(String) }
+    def name
+      collection.name.presence || 'No Title'
+    end
+
+    sig { returns(String) }
+    def spinner
```

Comment:
I wonder if this could move to the ApplicationComponent but I don't think it matters too much right now.

Reply:
I would not put it there, because this isn't something for every component.  If anything it could be its own component.